cvmfs/cvmfs.cc

```diff
@@ -2643,7 +2643,10 @@ static int Init(const loader::LoaderExports *loader_exports) {
     download::DownloadManager::kSetProxyBoth);
   g_external_download_ready = true;
   if (use_geo_api) {
-    cvmfs::external_download_manager_->ProbeGeo();
+    std::vector<std::string> host_chain;
```

Comment:
It shouldn't.  There's an early-exit in the case where the sorting is trivial (less than 2 servers).